### ISSUE TITLE
Do not cast message to string

### ISSUE THREAD

**Issue by david-zacharias** [NatLibFi/loglevel-message-prefix#5]
I have problem with dumping objects in logs when _loglevel-message-prefix_ is enabled. 

The result of `log.debug(action);` is `[11.1.2016 13:55:32 DEBUG]: [object Object]`.

How about changing line 149 from

``` JS

fn_method_raw('[' + prefix + ']: ' + message);
```

to 

``` JS
fn_method_raw('[' + prefix + ']: ', message);
```

so that your plugin does not cast the variable type of `message`?

**Comment by eskatos** [comment]
:+1:

**Comment by ghost** [comment]
Sorry for the late reply! This repository was erroneously not on my watch list.

I'll look into this issue soon.

**Comment by ghost** [comment]
Resolved with ddb65b80098ee9f3327acf5d3401a26df009f122.

Concatenation is done using JSON.stringify if the message is an object. This does not yield as pretty representation of the object as in the suggested implementation by @david-zacharias, but it's cross-compatible (console.log is not standard, AFAIK).
